ebuild-maintenance/git/text.xml

```diff
@@ -127,17 +127,18 @@ used. The valid uses of git include:
 <ul>
 <li>creating commits spanning multiple packages and/or multiple areas
 of the Gentoo repository (eclasses, licenses, profiles…),</li>
-<li>amending a commit created via <c>repoman commit</c> with additional files
-or fixups,</li>
-<li>combining multiple commits created via <c>repoman commit</c> using <c>git
-rebase</c>.</li>
+<li>amending a commit created via <c>repoman commit</c> or <c>pkgdev commit</c>
+with additional files or fixups,</li>
+<li>combining multiple commits created via <c>repoman commit</c> or
+<c>pkgdev commit</c> using <c>git rebase</c>.</li>
 </ul>
 
 <p>
-Whenever repoman is not used to commit, you need to manually verify all
-packages affected by the commit using <c>repoman full</c>. Since repoman
-is not aware of staged changes, please make sure that all files are included
-in the commit. Also when not using repoman, you must perform a manual sign-off
+Whenever <c>repoman</c> or <c>pkgdev</c> is not used to commit, you need to
+manually verify all packages affected by the commit using <c>repoman full</c> or
+<c>pkgcheck scan --commits</c>. Note that if using <c>repoman</c>, it is not
+aware of staged changes, so please ensure that all files are included in the
```

Comment:
```suggestion
aware of staged changes, so ensure that all files are included in the
```
I find this more guide-like than a request.

Reply:
Agreed.